bee-storage/bee-storage-test/src/message_id_to_metadata.rs

```diff
@@ -67,6 +69,20 @@ pub fn message_id_to_metadata_access<B: StorageBackend>(storage: &B) {
     assert_eq!(results.len(), 1);
     assert!(matches!(results.get(0), Some(Ok(Some(v))) if v == &metadata));
 
+    let milestone_index = MilestoneIndex(0);
```

Comment:
Can you do something like fetch it before, increase and check again ? Just in case the index was already 0 idk